cmd/cli/uninstall.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+/*
```

Comment:
Remove the commented code?

Reply:
Oh man I accidentally added that file. Thanks @allenlsy!